pkg/ui/workspaces/cluster-ui/src/jobs/util/jobOptions.tsx

```diff
@@ -95,7 +95,9 @@ export const statusOptions = [
   { value: JOB_STATUS_REVERT_FAILED, name: "Revert Failed" },
 ];
 
-const ALL_JOB_STATUSES = new Set(statusOptions.map(option => option.value));
+export const ALL_JOB_STATUSES = new Set(
```

Comment:
All this was for the test case, specifically the `it.each` to avoid importing all of the different statuses separately.

Reply:
Although it may be a good idea to do the separate imports anyway? I noticed that `ALL_JOB_STATUSES` includes a `"All"`, and may be the reason that it wasn't exported originally.

It does seem a bit odd that we have the statuses all declared as separate variables, I assume may just be an artifact of older code. Do we have any opinions here on whether `enum` vs `as const` is preferred?

Reply:
Oh this was my bad, I thought this was just a format change! I missed that export was added. Please ignore my original comment.

> I noticed that ALL_JOB_STATUSES includes a "All", and may be the reason that it wasn't exported originally.
It looks like it wasn't exported before because it was only used to check if a status is valid. I also question if an empty string is actually a valid status though... That seems like the null option for a dropdown.

>It does seem a bit odd that we have the statuses all declared as separate variables, I assume may just be an artifact of older code. Do we have any opinions here on whether enum vs as const is preferred?

If you were creating these job status values today, `enum` would be preferred, but if you want to refactor the usage, please do that in a separate PR. You'd have to change all the references which I think would be a bit sprawling 😄